README.md

```diff
@@ -49,6 +49,23 @@ How To Get
 Check the [Github Project](https://github.com/nealey/hdjd) for the canonical source repo.
 
 
+How to Build
+------------
+
+Install Dependencies:
+
+* Debian / Ubuntu
+
+ apt-get install libusb-1.0.0-dev libasound2-dev
```

Comment:
Please add 4 spaces to the beginning of this line so it will show up as preformatted text:

```markdown
    apt-get install libusb-1.0.0-dev libasound2-dev
```

Will render as:

    apt-get install libusb-1.0.0-dev libasound2-dev